woocommerce/woocommerce-gutenberg-products-block
# Make `Filters Products by Price` work with `Active Filters` block

When we render the PHP Classic template with the Filter Products by Price block we need to make sure the Active Filters block is also updated with the `min_price` and `max_price` values from the URL.

Note: clicking on **Clear All** on the `Active Filters` block won't actually clear the price on the slider. That's a known limitation of the `Active Filters` block and will be addressed on here: https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/6138

Fixes https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/6243

### Testing

- Add the `Filter Products by Price` block & `Active Filters` block to the **Shop** Page
- Open the **Shop** page
- Use the `Filter Products by Price` block, the page should reload with the filter params now being in the URL too.
- Check the `Active Filters` block shows the active price filter.

### Changelog

> Fix: Make Filters Products by Price work with Active Filters block for the PHP rendered Classic Template.
